src/stores/globalProductFilters.ts

```diff
@@ -0,0 +1,8 @@
+import { FilterBuilder, ConditionBuilder, DataValueFactory } from '@relewise/client';
+import contextStore from './context.store';
+
+export function globalProductRecommendationFilters(filterBuilder: FilterBuilder) {
```

Comment:
Vi bør vel kun skjule "Sold Out" produkter i anbefalinger.
Jeg synes at man stadig skal kunne fremsøge udsolgte produkter således at man kan demo den del.

Reply:
Den bliver kun brugt i recommendations, bemærk også navnet: `globalProductRecommendationFilters`

Reply:
Ahh yes fedt! Skal vi ikke rette det i selve check mark i settings? Det står der at den også gælder for search :)